website/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -euf -o pipefail
+gitbook build
+rm -rf ../docs
```

Comment:
same here?

Reply:
it is... because `/docs` is auto-generated and there's git to track history.